### DISCUSSION TITLE
Floodlight lights always ON since 9.22.0!!

### DISCUSSION THREAD

**Discussion by PaulmichlJ** [dgreif/ring#785] [Q&A] [answered]
Hallo, seit der Version 9.22.0 leuchten die Lichter DURCHGEHEN bei allen meinen ring Floodlight!
Manuelle deaktivierten wird ca alle 50 Minuten mit „Light on“ überschrieben!

Wer hat eine Lösung bzw. wie kann ich auf die funktionierende Vor-Version von 9.22.0 zurück??

**Comment by dgreif** [comment, marked as answer]
Duplicate of #786

**Comment by PaulmichlJ** [comment]
SORRY - new to GitHub . . .

Regards,
Jürgen

Sent from my mobile, sorry for shortened statement  and please ignore any orthographic mistakes ;-)

Mit bestem Gruß,
MMst. Jürgen M. Paulmichl, BSc MA
_____________________________________________________________________________

Leutenhofen 30
A-6914 Hohenweiler
Austria/Europe

Mobile:+43/699/10017769
EMail: ***@***.***

_______________________________________________________________________________
Disclaimer:
This email is confidential and intended solely for the use of the individual to whom it is addressed. Any views
or opinions presented are solely those of the author and do not necessarily represent those of his company.

If you are not the intended recipient, be advised that you have received this email in error and that any use, 
dissemination, forwarding, printing, or copying of this email is strictly prohibited.
Although the sender endeavours to maintain a computer virus free network, the sender does not warrant that
this transmission is virus-free and will not be liable for any damages resulting from any virus transmitted. 
Thank you.
PLEASE CONSIDER THE ENVIRONMENT BEFORE YOU PRINT THIS E-MAIL
Antes de Imprimir pense em sua responsabilidade e compromisso com o MEIO AMBIENT
